src/runtime/HalidePyTorchHelpers.h

```diff
@@ -59,12 +84,18 @@ AT_FORALL_SCALAR_TYPES_WITH_COMPLEX(HL_PT_DEFINE_TYPECHECK);
 
 #undef HL_PT_DEFINE_TYPECHECK
 
+#endif
```

Comment:
Nit: It'd be nice to see what this `#endif` corresponds to.

Reply:
Done